test/spectral/tests/enabled-rules.test.js

```diff
@@ -0,0 +1,522 @@
+const commandLineValidator = require('../../../src/cli-validator/runValidator');
+const { getCapturedText } = require('../../test-utils');
+
+describe('spectral - test enabled rules - Swagger 2', function() {
+  let consoleSpy;
+
+  beforeEach(() => {
+    consoleSpy = jest.spyOn(console, 'log').mockImplementation(() => {});
+  });
+
+  afterEach(() => {
+    consoleSpy.mockRestore();
+  });
+
+  it('test no-eval-in-markdown rule using mockFiles/swagger/enabled-rules.yml', async function() {
+    // set up mock user input
+    const program = {};
+    program.args = ['./test/spectral/mockFiles/swagger/enabled-rules.yml'];
+    program.default_mode = true;
+    program.print_validator_modules = true;
+
+    const exitCode = await commandLineValidator(program);
+    const capturedText = getCapturedText(consoleSpy.mock.calls);
+    const allOutput = capturedText.join('');
+    // Spectral should be the only validator module in the output
+    const count = (allOutput.match(/Validator:/g) || []).length;
+
+    expect(exitCode).toEqual(1);
+    expect(count).toEqual(2);
```

Comment:
I think I just don't understand this check but I expected this "count" to be 1

Reply:
Well I'm hoping to just test that Spectral is the only validator module that is included in the output (to test for possible duplicate rules). I guess the issue is, we print each validator module in errors/warnings:
```
errors

Validator: spectral
  Message :   `number_of_coins` property type should be integer
  Path    :   definitions.ModelWithBadExample.example.number_of_coins
  Line    :   177

warnings

Validator: spectral
```
So maybe I can think of a better way to test this..